helpers/testModel.spec.js

```diff
@@ -0,0 +1,39 @@
+const request = require('supertest');
+
+const db = require('../data/dbConfig.js');
+const dbModel = require('./testModel.js');
+
+describe('hobbits model', () => {
+    describe('insert()', () => {
+        afterEach(async () => {
+            await db('test').truncate()
+        })
```

Comment:
this is a cool method - just make sure you never use this in production - don't want to accidentally delete all your data when you run tests. This is why the cleanup method is nice